hat/app/org/hatdex/hat/utils/Mailer.scala

```diff
@@ -145,96 +147,90 @@ class HatMailerImpl @Inject() (
   def serverErrorNotify(
       request: RequestHeader,
       exception: UsefulException
-    )(implicit m: Messages): Future[Unit] = {
+    )(implicit m: Messages): Future[Unit] =
```

Comment:
@xavji is the removal of the `{` expected? Just checking to not add noise to the future PRs

Reply:
it is `scalafmt`, yes it is all good